metals/src/main/scala/scala/meta/internal/metals/InteractiveSemanticdbs.scala

```diff
@@ -55,7 +55,7 @@ final class InteractiveSemanticdbs(
   override def textDocument(source: AbsolutePath): TextDocumentLookup = {
 
     def shouldTryCalculateInteractiveSemanticdb = {
-      source.isDependencySource(workspace) || source.isSbt || buildTargets
```

Comment:
Right underneath this check we have `!source.isScalaFilename` and part of the `isScalaFilename` is the `isSbt` check. So this is redundant.

Reply:
I don't get this. It's failed twice in a row now in the sbt test... so it makes me think it's related to this, but I don't get why this would be needed as well here.

Reply:
It might be do to the fact that SBT files are in a build target and we don't calculate semanticdb for those. 🤔

Reply:
I'm not sure. I added the `source.isSbt` back in and now everything is happy again.